### DISCUSSION TITLE
Custom notifications

### DISCUSSION THREAD

**Discussion by miklosania** [Budibase/budibase#5234] [Help] [answered]
![msg](https://user-images.githubusercontent.com/89782874/161543226-50c90de6-24dc-469c-9932-03d9c1bcc179.jpg)

Hi Everyone,

I would like to ask if it is possible to customize Budibase notifications and pop-up alert boxes?

**Comment by melohagan** [comment, marked as answer]
Budibase uses Spectrum Toast: https://opensource.adobe.com/spectrum-css/toast.html

There is some customisation available through class names, but there are no immediate plans in the budibase UI to make use of this.  

It is a good feature idea though, and a couple of others have requested this.

**Reply by miklosania** [reply]
Thank you! I'll look forward to it.

**Comment by JayP718** [comment]
Agreed this is much needed. While the alerts are helpful for a developer it's a bit quirky for an end user.

**Reply by ryzam** [reply]
waiting custom message notification, really needed

**Reply by JayP718** [reply]
Check out this thread https://github.com/Budibase/budibase/issues/7371
